api/resources_portal/migrations/0004_auto_20200422_1753.py

```diff
@@ -0,0 +1,21 @@
+# Generated by Django 2.2.12 on 2020-04-22 17:53
+
+import django.contrib.postgres.fields
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("resources_portal", "0003_auto_20200422_1627"),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name="material",
+            name="organism",
+            field=django.contrib.postgres.fields.ArrayField(
+                base_field=models.TextField(), blank=True, null=True, size=None
+            ),
+        ),
+    ]
```

Comment:
Unless there's any reason a migration needs to be done in two parts I prefer if each PR only adds one migration.

Reply:
Done.